ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -102,6 +102,34 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-topology-console
+  cron: 0 */24 * * *
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
+    test:
+    - as: test
+      cli: latest
+      commands: ./test-prow-e2e.sh nightly-cypress topology
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+      timeout: 2h30m0s
+    workflow: ipi-gcp
+- as: e2e-pipeline-console
+  cron: 0 */24 * * *
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
+    test:
+    - as: test
+      cli: latest
+      commands: ./test-prow-e2e.sh nightly-cypress pipeline
```

Comment:
`pipeline` doesn't exist, should be `pipelines`

Reply:
Updated